assets/prometheus-operator/deployment.yaml

```diff
@@ -46,7 +46,7 @@ spec:
         resources:
           requests:
             cpu: 5m
-            memory: 60Mi
+            memory: 125Mi
```

Comment:
If I remember correctly, one of the replica of prometheus-operator was often using around 150MiB of memory. That said, wouldn't it be better to readjust the memory request of prometheus-operator to the value of the replica with the highest memory consumption which is 150Mi in this case?

Reply:
Good point :+1: Adjusted to 150Mi